contrib/cirrus/lib.sh

```diff
@@ -198,7 +198,7 @@ in_podman() {
                    --cgroupns=host \
                    "${envargs[@]}" \
                    -e BUILDAH_ISOLATION \
-                   -e STORAGE_DRIVER \
+                   -e STORAGE_DRIVER"${NESTED_STORAGE_DRIVER:+=${NESTED_STORAGE_DRIVER}}" \
```

Comment:
There used to be changes to `.cirrus.yml` that added `NESTED_STORAGE_DRIVER` to the matrix; I can’t see any current equivalent, was that dropped intentionally?